data/readreplica.py

```diff
@@ -112,20 +137,21 @@ def _select_database(cls):
         # Select the master DB if read replica support is not enabled.
         read_only_config = cls._read_only_config()
         if not read_only_config.read_replicas:
-            return cls._meta.database
+            return DoubleWriteWrapper( cls._meta.database, None, cls._meta.double_write_database)
```

Comment:
instead of wrapping every invocation with the `DoubleWriteWrapper` manybe only do it if `cls._meta.double_write_database` at one place

Reply:
this is planned in the cleanup.